src/coreclr/jit/valuenum.cpp

```diff
@@ -57,6 +57,8 @@ struct FloatTraits
         unsigned bits = 0xFFC00000u;
 #elif defined(TARGET_ARMARCH)
         unsigned           bits = 0x7FC00000u;
+#elif defined(TARGET_LOONGARCH64)
+        unsigned           bits = 0xFFC00000u;
```

Comment:
Looks like the same as TARGET_XARCH; merge ifdefs

Reply:
This is a mistake. I will modify it.
Thanks